h/cli/commands/publisher.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+
+import click
+
+from h import models
+
+
+@click.group()
+def publisher():
+    """Manage publisher accounts."""
+
+
+@publisher.command()
+@click.option('--name', prompt=True, help="The name of the publisher")
+@click.option('--authority', prompt=True, help="The authority (domain name) for the publisher")
+@click.pass_context
+def add(ctx, name, authority):
+    """
+    Create a new publisher account.
+
+    This consists of an authority, an admin user and the main group for the
+    publisher's annotations.
+    """
+    request = ctx.obj['bootstrap']()
+
+    # Create a new authority.
+    authclient = models.AuthClient(name=name, authority=authority)
```

Comment:
According to https://hyp.is/NnyczN2qEeayc1f9U4OdYw/click.pocoo.org/latest/arguments/ , string arguments are always unicode.